test/result/function/func_math_floor.result

```diff
@@ -135,11 +135,9 @@ select floor(-123.342124124214, -100);
 floor(-123.342124124214, -100)
 -1.0E100
 select floor(123.342, null);
-floor(123.342, null)
-null
+the second argument of the round function must be an int64 constant
```

Comment:
This is floor function, why the error message is for round function?

Reply:
yeah, the floor copy the error message. I fixed it.